lib/ClimaLandSimulations/src/Fluxnet/fluxnet_utilities/site_parameters.jl

```diff
@@ -0,0 +1,98 @@
+export site_parameters,
+    site_params, hetero_resp, soil, vangenuchten, κ, plant_hydraulics
+
+"""
+    site_parameters(;
+        params,
+        )
+
+Setup parameters for a site
+"""
+function site_parameters(site_ID)
+    default_params_dict = Dict(
+        "US-MOz" => ozark_default_params(),
+        #"US-Ha1" => harvard_default_params(), # need to add those once I create new harvard_default_params() etc
+        #"US-NR1" => niwotridge_default_params(),
+        #"US-Var" => vairaranch_default_params(),
+    )
+
+    if haskey(default_params_dict, site_ID)
+        return default_params_dict[site_ID]
+    else
+        println("This site_ID does not exist")
+    end
+end
+
+# The structs below are necessary and different from ClimaLand/src struct 
+# because of, for example, shared parameters (e.g., porosity in soil or soilco2)
+# or, another example, parameters calculated from other parameters (e.g., κ_solid)
+# the structs below only contain unique params, that are given as number (not computed FTwhere)
+
+struct hetero_resp # differs from SoilCO2ModelParameters because of porosity
```

Comment:
I'm still quite opposed to these structs since they re-define structs we already have. Maybe we can brainstorm other ways to fix the problems that they are addressing. future PR ok but this seems quite important

Reply:
Definitely! We need to find a way to use our src struct.